packages/yoastseo/spec/scoring/assessments/inclusiveLanguage/configuration/disabilityAssessmentsSpec.js

```diff
@@ -111,7 +111,17 @@ describe( "Disability assessments", function() {
 		} );
 	} );
 
-	it( "should not dumb if preceded by 'deaf and'.", () => {
+	it( "should target 'stupid'.", () => {
+		const assessment = new InclusiveLanguageAssessment( assessments.find( obj => obj.identifier === "stupid" ) );
+
+		const testSentence = "Stupid, that's just how they're acting.";
+		const mockPaper = new Paper( testSentence );
+		const mockResearcher = Factory.buildMockResearcher( [ testSentence ] );
+
+		expect( assessment.isApplicable( mockPaper, mockResearcher ) ).toBe( true );
```

Comment:
Is there any specific reason why you did not add the expected score, feedback en marks in the spec?

Except if there is something special about this assessment, I would suggest to stay consistent with all/most other assessments.

Reply:
The surrounding `it`'s in the spec didn't have them, so I didn't include them initially. But let's add them for good measure, can't hurt 👍